pontoon/base/static/js/fluent_interface.js

```diff
@@ -722,6 +722,12 @@ var Pontoon = (function (my) {
           }
         }
 
+        // Overcome bug in fluent-syntax 0.6.0 that incorrectly parses
+        // empty messages without trailing newlines as Messages instead of Junk
```

Comment:
This bug has been fixed in `fluent-syntax` 0.6.2. Do we still need this workaround?